x-team/xp-ui
# Add AccountConfirmation component

**Release Type:** *Non-breaking changes*

Fixes https://x-team-internal.atlassian.net/browse/XP-3596

## Description

Add `AccountConfirmation` component and upgrade `Modal` with white theme.

## Checklist


## Related PRs

branch | PR
------ | ------
`XP-3596-allow-users-to-logout-on-reg-form-and-jobs-pages` | [XP Registration](https://github.com/x-team/xp-registration/pull/159)

## Steps to Test or Reproduce

- Check `AccountConfirmation` at http://localhost:9001/?path=/story/ui-components-accountconfirmation--basic-usage
- Check Modal with white theme at http://localhost:9001/?path=/story/core-components-modal-states--white-theme

## Impacted Areas in Application

- `src/components/ui/AccountConfirmation.js`
- `src/components/ui/Modal.js`

## Screenshots

<img width="1333" alt="Screen Shot 2020-03-18 at 12 51 24" src="https://user-images.githubusercontent.com/131859/76979781-31452200-6917-11ea-9b33-6da029bf38eb.png">
